test/axes/timeAxisTests.ts

```diff
@@ -281,4 +282,15 @@ describe("TimeAxis", () => {
 
   });
 
+  it("height includes margin, label padding, and tick length", () => {
+    let svg = TestMethods.generateSVG(400, 400);
+    let xScale = new Plottable.Scales.Time();
+    let xAxis = new Plottable.Axes.Time(xScale, "bottom");
+    xAxis.margin(100);
+    xAxis.renderTo(svg);
```

Comment:
It seems like rather than `renderTo()`, it would be better to check this by `anchor()`ing the `Axis`, then calling `requestedSpace` to see that it asks for enough, right?